includes/admin/amp-wp-admin-styles.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+add_action('admin_head', 'amp_options_styles');
+function amp_options_styles() {
```

Comment:
We can use this hook instead: https://developer.wordpress.org/reference/hooks/admin_print_styles-hook_suffix/ 

```
add_action( 'admin_print_styles-' . $this->menu_slug, array( $this, 'amp_options_styles' ) );
```